src/orquestra/sdk/_base/cli/_dorq/_ui/_errors.py

```diff
@@ -89,10 +89,20 @@ def _(e: exceptions.WorkflowRunNotFinished) -> ResponseStatusCode:
 @pretty_print_exception.register
 def _(e: ConnectionError) -> ResponseStatusCode:
     _print_traceback(e)
-    click.echo("Unable to connect to Ray.")
+    click.echo(f"{e}")
```

Comment:
**question,non-blocking**: what will see the user when the CLI can't connect to Ray?

Reply:
Whatever the exception Ray provided. I had changed this cause a ConnectionError could have come from the HTTP requests.